src/interactable_setPlayerLayer.c

```diff
@@ -0,0 +1,74 @@
+#include "global.h"
```

Comment:
Would prefer file name of `interactable_set_player_layer`

Or, if we could consider it, the name could be `PlayerLayerToggle` or `PlayerLayerTrigger`

Reply:
Either way, the name should only be snake case in the filename